lib/plug/debugger.ex

```diff
@@ -549,4 +549,42 @@ defmodule Plug.Debugger do
   defp maybe_merge_dark_styles(style, default_dark_style) do
     Map.put(style, :dark, default_dark_style)
   end
+
+  defp maybe_autolink(message) do
+    splitted = Regex.split(~r/`([^`]+)`/, message, include_captures: true, trim: true)
+
+    Enum.map(splitted, &maybe_format_function_reference/1)
+    |> IO.iodata_to_binary()
+  end
+
+  defp maybe_format_function_reference("`" <> reference = text) do
+    reference = String.trim_trailing(reference, "`")
+
+    with true <- function_reference?(reference),
+         {:ok, m, f, a} <- get_mfa(reference),
+         url when is_binary(url) <- get_doc(m, f, a, Application.get_application(m)) do
+      ~s[<a href="#{url}" target="_blank">`#{h(reference)}`</a>]
+    else
+      _ -> h(text)
+    end
+  end
+
+  defp maybe_format_function_reference(text), do: h(text)
+
+  defp function_reference?(text) do
+    Regex.match?(~r/^[A-Z][A-Za-z0-9_.]+\.[a-z][A-Za-z0-9_!?]*\/\d+$/, text)
+  end
+
+  def get_mfa(capture) do
+    with [function_path, arity] <- String.split(capture, "/"),
```

Comment:
Here is how this is done in ExDoc: https://github.com/elixir-lang/ex_doc/blob/main/lib/ex_doc/language/elixir.ex#L311

It has to deal with a bunch of other cases, like `left..right` and local functions, but it should have a more robust code for parsing both modules and functions. :)

Reply:
Now that I think about it, I wonder if there could be security implications here. If we parse the error message and we are converting strings to atoms, an error message that is based from the user input can now generate atoms. I think we should keep the current parsing code, and not use the one from ExDoc, and also use only `String.to_existing_atom`.

Reply:
I've seen the ExDoc code, but I don't think that we need all those cases here and just focus on the low hanging fruit. To handle local functions, we could look into the stacktrace, but I'm not sure if that's worth it.

If we use `String.to_existing_atom` we should also add a rescue and just use the text as is if it fails, right?